utils/mime.go

```diff
@@ -36,6 +36,9 @@ const (
 
 	// MimeTypeDefault used if mimetype cannot be inferred.
 	MimeTypeDefault = "application/octet-stream"
+
+	// MimeTypeEmpty used for delegating the choice of MimeType to the server .
+	MimeTypeEmpty = ""
```

Comment:
While the variable is technically the same as "", it will be easier for future developers to understand that it's just the empty string by not using a named variable

Reply:
yeah I'd remove this

Reply:
done